File: scala-gatling/src/gatling/scala/org/openapitools/client/model/ResponseTimeFilterPostcodeDistrictsResult.scala

package org.openapitools.client.model


case class ResponseTimeFilterPostcodeDistrictsResult (
    _searchId: String,
    _districts: List[ResponseTimeFilterPostcodeDistrict]
)
object ResponseTimeFilterPostcodeDistrictsResult {
    def toStringBody(var_searchId: Object, var_districts: Object) =
        s"""
        | {
        | "searchId":$var_searchId,"districts":$var_districts
        | }
        """.stripMargin
}

File: dart-jaguar/lib/model/response_time_filter_postcode_districts_result.dart
import 'package:jaguar_serializer/jaguar_serializer.dart';


import 'package:openapi/model/response_time_filter_postcode_district.dart';

part 'response_time_filter_postcode_districts_result.jser.dart';

class ResponseTimeFilterPostcodeDistrictsResult {
  
  @Alias('search_id', isNullable: false,  )
  final String searchId;
  
  @Alias('districts', isNullable: false,  )
  final List<ResponseTimeFilterPostcodeDistrict> districts;
  

  ResponseTimeFilterPostcodeDistrictsResult(
      

{
    
     this.searchId = null,  
     this.districts = const [] 
    }
  );

  @override
  String toString() {
    return 'ResponseTimeFilterPostcodeDistrictsResult[searchId=$searchId, districts=$districts, ]';
  }
}

@GenSerializer(nullableFields: true)
class ResponseTimeFilterPostcodeDistrictsResultSerializer extends Serializer<ResponseTimeFilterPostcodeDistrictsResult> with _$ResponseTimeFilterPostcodeDistrictsResultSerializer {

}

